src/builder/create_embed.rs

```diff
@@ -233,10 +225,12 @@ impl CreateEmbed {
     /// impl EventHandler for Handler {
     ///     async fn message(&self, context: Context, mut msg: Message) {
     ///         if msg.content == "~embed" {
+    ///             let timestamp: Timestamp =
+    ///                 "2004-06-08T16:04:23".parse().expect("Invalid timestamp!");
```

Comment:
```suggestion
    ///                 "2004-06-08T16:04:23Z".parse().expect("Invalid timestamp!");
```

Missing a UTC offset at the end of the timestamp which causes it to fail parsing. Alternatively you could put `+04:00` or whatever.

Reply:
Haha, that means it was broken beforehand and panicked if actually used, nice catch